src/exchange_adapters/coinbase.ts

```diff
@@ -5,7 +5,7 @@ import { CeloContract } from '@celo/contractkit'
 
 export class CoinbaseAdapter extends BaseExchangeAdapter implements ExchangeAdapter {
   baseApiUrl = 'https://api.pro.coinbase.com'
-  // Cloudflare Inc ECC CA-3
+  // pro.coinbase.com - validity not after: 11/05/2024, 01:59:59 CEST
```

Comment:
looks like you updated the validity here but not the certificate. For pro.coinbase.com I get `7B:F6....3B:39`